cmd/access/main.go

```diff
@@ -91,6 +91,7 @@ func main() {
 			flags.StringVarP(&rpcConf.HistoricalAccessAddrs, "historical-access-addr", "", "", "comma separated rpc addresses for historical access nodes")
 			flags.DurationVar(&rpcConf.CollectionClientTimeout, "collection-client-timeout", 3*time.Second, "grpc client timeout for a collection node")
 			flags.DurationVar(&rpcConf.ExecutionClientTimeout, "execution-client-timeout", 3*time.Second, "grpc client timeout for an execution node")
+			flags.UintVar(&rpcConf.MaxHeightRange, "rpc-max-height-range", 100, "maximum size for height range requests")
```

Comment:
I know some people are doing close the this 100 range currently, lets put this default at 200 so they have some room.